pkg/boltz/ws.go

```diff
@@ -201,6 +219,11 @@ func (boltz *Websocket) subscribe(swapIds []string) error {
 }
 
 func (boltz *Websocket) Subscribe(swapIds []string) error {
+	if boltz.state == disconnected {
+		if err := boltz.Connect(); err != nil {
```

Comment:
There should be a check here for `len(swapIds) > 0` to make sure that we don't open and close the websocket instantly afterwards